examples/seismic/acoustic/operators.py

```diff
@@ -44,7 +44,7 @@ def iso_stencil(field, m, s, damp, kernel, **kwargs):
     # Add dampening field according to the propagation direction
     eq += damp * field.dt if kwargs.get('forward', True) else -damp * field.dt
     # Solve the symbolic equation for the field to be updated
-    eq_time = solve(eq, next, rational=False, simplify=False)[0]
+    eq_time = solve(eq, next, rational=False, simplify=False)
```

Comment:
do we still need rational/simplify?

Reply:
yes

Reply:
but aren't they automatically applied by our own ``solve``?

Reply:
My bad you're right